frontend/src/common/components/naviconsidebar/NavIconSideBar.tsx

```diff
@@ -11,13 +13,15 @@ interface NavIconSideBarProps {
  * Displays a sidebar with NavIcon buttons as children
  */
 export const NavIconSideBar = (props: NavIconSideBarProps) => {
-  const { children } = props;
+  const { children } = props
   const { user } = useAuth()
   const isIdir = user?.profile?.identity_provider === IDPS.IDIR
+  const { idirUserDetails } = useContext(OnRouteBCContext)
+  const isEofficer = idirUserDetails?.userAuthGroup === USER_AUTH_GROUP.EOFFICER
 
   return (
     <div className="nav-icon-side-bar">
-      {isIdir && children}
+      {isIdir && !isEofficer && children}
```

Comment:
This is hiding the navigation bar from PPC_CLERK and SYS ADMIN users. Just checking for idir should have worked already?
Refer: https://github.com/bcgov/onRouteBCSpecification/blob/main/Staff%20Global%20Navigation.feature#L3

Reply:
Yeah when I assign myself an `EOFFICER` role, I can see it. That's confusing how it would hide it also from `PPC_CLERK` and Sys Admins 🤔